app/concerns/translation_controller_concern.rb

```diff
@@ -31,4 +32,8 @@ def translation_locale
   def notice_message(action)
     %{#{translation_locale.english_language_name} translation for "#{translated_item_name}" #{action}.}
   end
+
+  def put_translation_async(model, locale)
+    PublishingApiDraftWorker.perform_async_in_queue(nil, model.class.name, model.id, nil, locale)
```

Comment:
is it possible to do this through the [`PublishingApi`](https://github.com/alphagov/whitehall/blob/master/lib/whitehall/publishing_api.rb) class, to keep that class as the choke point for all Publishing API interaction?